target/linux/ramips/image/mt7620.mk

```diff
@@ -290,6 +290,19 @@ define Device/ex3700-ex3800
 endef
 TARGET_DEVICES += ex3700-ex3800
 
+define Device/fon2601
```

Comment:
Please try: ```define Device/fon_fon2601```

This should also make the SUPPORTED_DEVICES entry unnecessary.

Reply:
changed at def4fb1